### PR TITLE
Malawi style tweaks

### PR DESCRIPTION
**Pull request by projectmushroom**

### CODE REVIEW

File: gem/client/css/themes/malawi.scss [hunk 1]
```diff
@@ -132,6 +135,13 @@ a {
       font-family: $font-family-zathu;
       font-size: 12px;
       font-weight: normal;
+      
+      a {
+        font-family: $font-family-default;
+        font-weight: bold;
+        color: $colorOrange;
+        font-size: $font-small;
```
Comment (Mitso) [inline gem/client/css/themes/malawi.scss]: This might be duplicated a tag inside .section-title {---}

File: gem/client/css/themes/malawi.scss [hunk 2]
```diff
@@ -22,7 +22,7 @@ $colorGrey: #dadada;  // grey
 $font-family-zathu: 'zathu', Arial, sans-serif;
 $font-family-default: Arial, sans-serif;
 
-html,body, .article {
+html,body, .article, {
```
Comment (Mitso) [inline gem/client/css/themes/malawi.scss]: Please remove  .article, comma typo

**Comment by Mitso** [comment]
Just few typo otherwise 👍

File: gem/client/css/themes/malawi.scss (hunk 1 above)
Comment (projectmushroom) [inline gem/client/css/themes/malawi.scss]: looks like it, but its not, first one sits inside  .template-main #article-hero... 😀

**Pull request merged**